relnotes/reverse.feature.md

```diff
@@ -0,0 +1,5 @@
+* `ocean.core.array.Iteration`
+
+  New module containing single function for now - `reverse`, which allows to
+  iterate given slice in reverse order without any allocations (similar to
+  built-in `foreach_reverse`).
```

Comment:
Yeah... I dunno. Whatever you think of `foreach_reverse`, it exists. My feeling is, why not use it? I don't see a great deal of sense in replicating built-in language features in library code, just out of some principle.